Introduction-to-Grails-Development-with-Intellij/index.md

```diff
@@ -0,0 +1,293 @@
+### Getting Started with Grails Java Framework
+
+In recent years, many developers are using frameworks when coming up with websites. Frameworks have assisted in making sure there is  faster application deployment and also coming up with quality code. Java supports multiplatform including a desktop application, web application, mobile app development, and many more. In the web application, it can be embedded with Javascript, HTML, CSS, and many front end applications.
+
+### Introduction
+
+**Grails** is Java and Groovy framework developing agile web applications. Grails implements MVCS (Model, View, and Controller) design pattern.
+
+Grails is superb because it makes developers concentrate on application requirements and use less time in configuring the framework. Grails provides tools for development and it is build based on tools like  Quarts, Hibernate, Spring and uses Gradle for library management.
+
+There are many Grails framework version being built which has the support of different libraries and tools
+
+### Prerequisites
+
+- IntelliJ integrated development environment(IDE)
+- Knowledge in Java development
+- Java development kit(JDK) installed in your computer/laptop to help in application development
```

Comment:
Consider providing download links to JDK and IntelliJ. Specify the Java version i.e. Java 11.

Reply:
solved